app/src/main/java/com/mapbox/maps/testapp/examples/MapViewCustomizationActivity.kt

```diff
@@ -18,16 +21,26 @@ import kotlinx.android.synthetic.main.activity_map_view_customization.*
 class MapViewCustomizationActivity : AppCompatActivity() {
 
   private lateinit var customMapView: MapView
+  // Users should keep a reference to the customised tileStore instance (if there's any)
+  private val tileStore by lazy {
+    TileStore.create().also {
+      // Users need to make sure the custom TileStore is initialised properly with valid access token
+      it.setOption(
+        TileStoreOptions.MAPBOX_ACCESS_TOKEN,
+        TileDataDomain.MAPS,
+        Value(getString(R.string.mapbox_access_token))
```

Comment:
Can we get token from ResourceOptionsManager?

Reply:
good catch